iOSClient/Menu/AppDelegate+Menu.swift

```diff
@@ -134,7 +134,7 @@ extension AppDelegate {
         //if !isDirectoryE2EE && CCUtility.isEnd(toEndEnabled: appDelegate.account) {
             actions.append(
                 NCMenuAction(title: NSLocalizedString("_create_folder_e2ee_", comment: ""),
-                             icon: UIImage(named: "folderEncrypted")!.image(color: NCBrandColor.shared.brandElement, size: 50),
+                             icon: UIImage(named: "encryptedfolder")!.image(color: NCBrandColor.shared.iconColor, size: 50),
```

Comment:
Instead of creating new e2ee folder icon modify the existing one. This will reduce our customization at code level.